go/client.go

```diff
@@ -356,6 +358,14 @@ func (s *ClientState) Run() error {
 	return nil
 }
 
+func (s *ClientState) closeRedisConnections() {
+	for _, rs := range s.redisSystems {
+		if rs.currentMaster != nil {
+			rs.currentMaster.Close()
+		}
```

Comment:
Nitpick: This could be a method on `RedisSystem`, so that we could just write `rs.Close()` here instead..

Reply:
yes. it could. it's what I had first. but then I thought: this is an unnecessary abstraction.

Reply:
there's also rs.currentMaster.IsMaster() and rs.currentMaster.Role() ...

I would need to hide all of those.

Reply:
to make it consistent.

Reply:
Hehe, ok. It's all very little code and just in a single file, so the value of changing it is not that big. I am fine with it as it is.